AI/MIGraphX/Quantization/Readme.md

```diff
@@ -0,0 +1,112 @@
+
+There are three main methods for computing quantization parameters:
+
+- Dynamic Quantization:
+    - Model weights are pre-quantized , input/activation quantization parameters are computed dynamically at runtime
+        
+- Static Post Training Quantization (PTQ):
+    - Quantization parameters are computed via calibration. Calibration involves calculating statistical attributes for relevant model nodes using provided sample input data
+        
+- Static Quantization Aware Training (QAT):
+  - Quantization parameters are calibrated during the training process
+
+**Note**: All three of these techniques are supported by PyTorch (at least in a prototype form), and so the examples leverage PyTorch's quantization APIs to perform this step.
+
+### Converting and executing the quantized model
+As of the latest PyTorch release, there is no support for executing quantized models on GPUs directly through the framework. To execute these quantized models, use AMD's graph optimizer, MIGraphX, which is built using the ROCm stack. The [torch_migraphx](https://github.com/ROCmSoftwarePlatform/torch_migraphx) library provides a friendly interface for optimizing PyTorch models using the MIGraphX graph optimizer. 
+
+The examples show how to use this library to convert and execute PyTorch quantized models on GPUs using MIGraphX.
+
+## Torch-MIGraphX
+
+Torch-MIGraphX integrates AMD's graph inference engine with the PyTorch ecosystem. It provides a `mgx_module` object that may be invoked in the same manner as any other torch module, but utilizes the MIGraphX inference engine internally. 
+
+This library currently supports two paths for lowering:
+
+- FX Tracing: Uses tracing API provided by the `torch.fx` library.
+  
+- Dynamo Backend: Importing torch_migraphx automatically registers the "migraphx" backend that can be used with the `torch.compile` API.
+
+
+## Getting started
+### Docker
+The simplest and recommended way to get started is using the provided Dockerfile.
+
+1. Build using:
+```
+./build_image.sh
+```
+2. Start container using:
+```
+sudo docker run -it --network=host --device=/dev/kfd --device=/dev/dri --group-add=video --ipc=host --cap-add=SYS_PTRACE --security-opt seccomp=unconfined torch_migraphx
+```
+
+The default Dockerfile builds on the nightly pytorch container and installs the latest source version of MIGraphX and torch_migraphx. For more builds, refer to the Docker directory.
+
+### Installing from source
+
+1. Install Prerequisites:
+
+- [PyTorch (ROCM version)](https://rocm.docs.amd.com/projects/install-on-linux/en/develop/how-to/3rd-party/pytorch-install.html#using-a-wheels-package)
+- [MIGraphX](https://github.com/ROCm/AMDMIGraphX?tab=readme-ov-file#installing-from-binaries)
+
+2. Build and install from source
+```
+git clone https://github.com/ROCmSoftwarePlatform/torch_migraphx.git
+cd ./torch_migraphx/py
+export TORCH_CMAKE_PATH=$(python -c "import torch; print(torch.utils.cmake_prefix_path)")
+pip install .
+```
+
```

Comment:
Even if the decision is to leave in the pasted installation instructions, the content below is probably not necessary since the examples in this repo themselves will show the user how to use the provided APIs in this context.

Reply:
@shivadbhavsar I added a link to the Torch_MIGraphX page.